client/src/main/java/com/flipkart/flux/client/runtime/FluxHttpClient.java

```diff
@@ -0,0 +1,6 @@
+package com.flipkart.flux.client.runtime;
+
+public interface FluxHttpClient {
+
+    void postOverHttp(Object dataToPost, String pathSuffix);
```

Comment:
Will the postOverHttp be used by the task runtime? or does the flux runtime connector get used by the runtime?